packages/core/schematics/migrations.json

```
@@ -9,6 +9,11 @@
       "version": "16.0.0",
       "description": "In Angular version 15.2, the guard and resolver interfaces (CanActivate, Resolve, etc) were deprecated. This migration removes imports and 'implements' clauses that contain them.",
       "factory": "./migrations/guard-and-resolve-interfaces/bundle"
+    },
+    "migration-v16-transfer-state": {
+      "version": "16.0.0",
+      "description": "Since Angular v16, `TransferState`, `makeStateKey`, `StateKey` has moved to the core package.",
```

NIT, the description should be what the migration does.

```suggestion
      "description": "Replace `TransferState`, `makeStateKey`, `StateKey` imports from `@angular/platform-browser` to `@angular/core`.",
```